cassiopeia.go

```diff
@@ -6,18 +7,30 @@ import (
 	"regexp"
 )
 
+// I guest this is the main package of this service right?
+// so, every interface needed in this package should be declared here(e.g blogger and halloffame)
+// we also need to create interface that will be implemented by Cassiopeia
+
+// this package shuld be implemented by cassiopeia, the intereface name is my random thought so i dont realy like it.
+// every function on Cassiopeia shoud be listed here first
+type ProclubMemberHallofFame interface {
```

Comment:
Just use `Cassiopeia` instead. No?

Reply:
i always use the service name as struct tho, but it is up to you

Reply:
Any consideration?